web/libs/editor/src/components/App/ViewAll.tsx

```diff
@@ -0,0 +1,49 @@
+import clsx from "clsx";
+import { ff } from "@humansignal/core";
+import { usePersistentState } from "@humansignal/core/lib/hooks/usePersistentState";
+import type { MSTAnnotation, MSTStore } from "../../stores/types";
+import Summary from "../Summary/Summary";
+import Grid from "./Grid";
+
+import styles from "./ViewAll.module.scss";
+
+type Props = {
+  store: MSTStore;
+  annotations: MSTAnnotation[];
+  root: any;
+};
+
+const Tab = ({ title, active, onSelect }: { title: string; active: boolean; onSelect: () => void }) => {
+  return (
+    <div className={clsx(styles.tab, {[styles.active]: active })} onClick={onSelect}>
+      {title}
+    </div>
+  );
+};
+
+export const ViewAll = ({ store, annotations, root }: Props) => {
+  const [tab, setTab] = usePersistentState<"summary" | "compare">("view-all-tab", "summary");
+
+  if (ff.isActive(ff.FF_SUMMARY)) {
```

Comment:
Should there also be a check that it is not standalone mode? I mean at least `window.DM` check, but maybe there is a better way.